docs/sources/architecture/configuration.md

```diff
@@ -29,6 +29,13 @@ Configuration descriptions must be placed as XML file(s) (with the ending `.xml`
         http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
 
   <config-description uri="{binding|thing-type|bridge-type|channel-type|any_other}://bindingID:...">
+    <group name="String">
```

Comment:
Ok - you're right. It doesn't matter to the reader, so you can put it wherever you like and it reads in ok, but the schema wants it in a specific order... Do you prefer it before or after the parameters? To me, it seems best to put them first, but I'm not too bothered...